README.md

```diff
@@ -168,7 +168,9 @@ Perhaps with additional research, a workaround can be conceptualized that does n
 
 To run end-to-end tests, you must first have built the extension (`npm run build`).
 Then the Playwright tests can be run with `npx playwright test` for headless mode.
-If you want to run the tests interactively, you can use `npx playwright test --ui`.
+To mock the network requests from the service worker an (currently experimental) Playwright environment
+variable must be set: PW_EXPERIMENTAL_SERVICE_WORKER_NETWORK_EVENTS=1.
+This is automatically set when running the tests with `npm run test`.
```

Comment:
I want to see something like this actually:

> To run end-to-end tests, you must first have built the extension (`npm run build`).
Then you run the tests via `npm test`.
If you want to run the tests interactively, you use `npm run test:ui`.

There is no need to add the details of the ENV parameter and what command the scripts execute. If people want to know they can check the `package.json`.